src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -10974,7 +10959,7 @@ public void RenderMode_With_Nullable_Receiver()
                 {
                     public class RenderModeContainer
                     {
-                        public Microsoft.AspNetCore.Components.IComponentRenderMode RenderMode => Microsoft.AspNetCore.Components.Web.RenderMode.Server;
+                        public Microsoft.AspNetCore.Components.IComponentRenderMode RenderMode => Microsoft.AspNetCore.Components.Web.RenderMode.InteractiveServer;
```

Comment:
Why did this need to change?

Reply:
Our shims are outdated - the member was called `Server` in some .NET previews, but in .NET 8 it shipped as `InteractiveServer`. I have updated the tests so they still compile correctly.

Reply:
That's wild! I'm glad that moving to this package rather than maintaining our own shims catches this.

Reply:
Yeah that is very wild.